### PR TITLE
correcting the docs for accessing static artifacts in a plugin

### PR DESCRIPTION
**Pull request by kensipe**

### CODE REVIEW

**Comment by kensipe** [comment]
the docs were incorrect... the solution has been tested

File: http/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Http Example Plugin
 
-This http handler plugin allows the handling of basic http request/response cycles. 
+This http handler plugin allows the handling of basic http request/response cycles.
 
 ## Usage
 
 See the [plugin configuration file](https://github.com/mesosphere/marathon-example-plugins/blob/master/http/src/main/resources/mesosphere/marathon/example/plugin/http/plugin-conf.json).
 The WebHarHandler will serve static content that is placed under the /META-INF/resources/webjars/static folder.
```
Comment (ichernetsky) [inline http/README.md]: `WebJarHandler`?

Comment (kensipe) [inline http/README.md]: it is one of the exposed plugins... I'm really glad you asked... as it was misspelled

File: http/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-The WebHarHandler will serve static content that is placed under the /META-INF/resources/webjars/static folder.
-It will serve the content via /static/: http://localhost:8080/v2/plugins/static/hello.txt
+Start up Marathon according to the example project [README.md](../README.md) file, using the [plugin configuration file](https://github.com/mesosphere/marathon-example-plugins/blob/master/http/src/main/resources/mesosphere/marathon/example/plugin/http/plugin-conf.json) configuration file.  This plugin example implements the [HttpRequestHandlerBase](https://github.com/mesosphere/marathon/blob/master/plugin-interface/src/main/scala/mesosphere/marathon/plugin/http/HttpRequestHandlerBase.scala) plugin with the [WebJarHandler](https://github.com/mesosphere/marathon-example-plugins/blob/master/http/src/main/scala/mesosphere/marathon/example/plugin/http/WebJarHandler.scala).   The implementation of the WebJarHandler class will serve static content that is placed under the [/META-INF/resources/webjars/static](https://github.com/mesosphere/marathon-example-plugins/blob/master/http/src/main/scala/mesosphere/marathon/example/plugin/http/WebJarHandler.scala#L11) folder.
```
Comment (ichernetsky) [inline http/README.md]: `WebJarHandler.scala).   Th` — redundant spaces. Also, this line is too huge.

File: http/README.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+
+It will serve the contents via: http://localhost:8080/v2/plugins/webjar/hello.txt where webjar is it's pluginid defined in the [plugin configuration](https://github.com/mesosphere/marathon-example-plugins/blob/master/http/src/main/resources/mesosphere/marathon/example/plugin/http/plugin-conf.json#L3).
```
Comment (ichernetsky) [inline http/README.md]: Quite a long line as well.

Comment (kensipe) [inline http/README.md]: it looks like a long line... but when markdown is rendered it doesn't seem like it

**Pull request merged**